src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -229,38 +237,47 @@ final class LazyList[+A] private(private[this] var lazyState: () => LazyList.Sta
   @inline private[this] def knownIsEmpty: Boolean = stateEvaluated && (isEmpty: @inline)
   @inline private def knownNonEmpty: Boolean = stateEvaluated && !(isEmpty: @inline)
 
-  /**
-    * Force the lazy list to evaluate all the elements. If it is a circular lazy list then the cycle is detected
-    * and the evaluation of further elements stops.
-    * Eg :- {{{
-    *   val ring: LazyList[Int] = 1 #:: 2 #:: 3 #:: ring
-    *   ring.force
-    *   ring.toString
+  /** Evaluates all undefined elements of the lazy list.
+    *
+    * This method detects cycles in lazy lists, and stops after all
+    * elements of the cycle are evaluated. For example:
+    *
+    * {{{
+    * val ring: LazyList[Int] = 1 #:: 2 #:: 3 #:: ring
+    * ring.force
+    * ring.toString
+    *
+    * // prints
+    * //
+    * // LazyList(1, 2, 3, ...)
     * }}}
-    * gives 'LazyList(1, 2, 3, ...)'
     *
-    * @return The lazy list that called the function.
+    * $willNotTerminateInf
```

Comment:
Do we need something more specific than that? Cyclic lazy lists *are* infinite, but forcing them does terminate.

Reply:
I can clone the message and clarify it, yeah